src/test/java/io/skai/okta/internshipstreamingserviceparserxml/services/converter/EpisodeAdditionalInfoConverterTest.java

```diff
@@ -0,0 +1,60 @@
+package io.skai.okta.internshipstreamingserviceparserxml.services.converter;
+
+import io.skai.okta.internshipstreamingserviceparserxml.dto.EpisodeAdditionalInfo;
+import io.skai.okta.internshipstreamingserviceparserxml.dto.ImdbJsonItem;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@ExtendWith(MockitoExtension.class)
+class EpisodeAdditionalInfoConverterTest {
+
+    @InjectMocks
+    EpisodeAdditionalInfoConverter converter;
+
+    private ImdbJsonItem item;
+    private static final Pattern ID_PATTERN = Pattern.compile("(?<=t{2})\\d+");
+
+    @BeforeEach
+    void setUp() {
+        item = ImdbJsonItem.builder()
+                           .imdbID("tt1480055")
+                           .seriesID("tt0944947")
+                           .imdbRating("9.1")
+                           .imdbVotes("44868")
+                           .runtime("62 min")
+                           .build();
+    }
+
+    @Test
+    void testConvertAdditionalInfoShouldBeNotNull() {
+        assertNotNull(converter.convert(9379992, item));
```

Comment:
this test is redundant as you have testConvertAdditionalInfoShouldBeSuccessful which checks the same thing + actual result

Reply:
i should remove this assertion fully or move it in testConvertAdditionalInfoShouldBeSuccessful()?